styles.scss

```diff
@@ -642,7 +642,6 @@ q {
           white-space: nowrap;
           overflow: hidden;
           text-overflow: ellipsis;
-          max-width: 320px;
```

Comment:
This was to prevent exaggeratedly wide columns like this:

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/e8ed7e81-db6a-473c-a79f-14efc9e20d41)

Reply:
yup I guess it was, but here's what [Laurent asked](https://mattermost.incubateur.net/fabnum-mte/pl/kszcyzrzu3nixj6h4kcu8zpjzr) : `Possible de ne pas tronquer la dernière colonne ?`

Reply:
Maybe we should show him this capture and ask him what to do?

Reply:
Maybe what we want is a parameter so we can specify the kind of column we want? narrow/wide, and maybe alignment (center, left, right) as I see we're not always consistent especially with numbers

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/fcf99062-355b-41a5-a919-31b6596e5b76)

My take would be to always right-align numbers.

Reply:
Sorry if this is outside of scope, we can probably file another card and I can work on this on my spare time